templates/flutter/lib/src/client_browser.dart.twig

```diff
@@ -214,7 +214,7 @@ class ClientBrowser extends ClientBase with ClientMixin {
   Future webAuth(Uri url, {String? callbackUrlScheme}) {
   return FlutterWebAuth2.authenticate(
       url: url.toString(),
-      callbackUrlScheme: callbackUrlScheme ?? "appwrite-callback-" + config['project']!,
```

Comment:
This would break the previous PR that attempts to make our flutter SDK work for windows auth

Reply:
Nope, this is only on `client_browser` which is only used on Flutter web. Other platform uses `client_io` which is still intact.

Reply:
Yes I agree we should remove it in client_browser